docs/meshstack.emergency-users.md

```diff
@@ -0,0 +1,98 @@
+---
+id: meshstack.emergency-users
+title: Emergency Users
+---
+
+meshStack manages access to cloud platforms, projects and resources. In cases when urgent intervention is required and users with regular access permissions are not available there must be a defined process to securely access projects and their associated platform resources. This page outlines step-by-step procedures that meshStack operators can use as the basis for their own emergency procedures. Depending on your organization's requirements, these procedures can be augmented with additional organisational or technical procedures.
```

Comment:
```suggestion
meshStack manages access to cloud platforms, projects and resources. In case urgent intervention is required by someone without regular access permissions there must be a defined process to securely access projects and associated platform resources. This page outlines step-by-step procedures that meshStack operators can use as the basis for their own emergency procedures. Depending on your organization's requirements, these procedures can be augmented with additional organisational or technical procedures.
```